CNAME

```diff
@@ -0,0 +1 @@
+python-sdk.meilisearch.com/
```

Comment:
One thing I just though of, wouldn't the `CNAME` file go in the docs folder instead of the root directory?

Reply:
From my understanding of the [Github docs](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site), it should be at the root of the repository.

Reply:
I'll go ahead and merge. The worst thing I can think that would happen is the CNAME doesn't get picked up and then we just need a new PR to move the file.